Libraries/ReactNative/YellowBox.js

```diff
@@ -69,10 +69,13 @@ if (__DEV__) {
     warn.apply(console, arguments);
     updateWarningMap.apply(null, arguments);
   };
+<<<<<<< HEAD
```

Comment:
@vjeux  My last fork was 5 days ago, so I try to remote update to actual base, but anyway I still have unresolved 2 conflicts. What would you recommend?